File: PrintN_times.java
// Question:
// Input: N = 3
// Output: Ashish Ashish Ashish 
// Explanation: Name is printed 3 times.

class Show
{   
    static int i = 0;
    public static void Display()
    {
        if(i <= 3)
        {
            System.out.println("Ashish");
            i++;

            Display();
        }
    }
}

class PrintN_times
{
    public static void main(String A[]) 
    {
        Show.Display();
    }
}

File: PrintN_times.cpp
// Question:
// Input: N = 3
// Output: Ashish Ashish Ashish 
// Explanation: Name is printed 3 times.


#include<iostream>
using namespace std;

void Display()
{
    static int i = 0;

    if(i < 3)
    {
        cout<<"Ashish\n";
        i++;
        Display();
    }
}


int main()
{   
    Display();

    return 0;
}